DemoApp/Assets/Scripts/HomeScreen.cs

```diff
@@ -95,10 +95,10 @@ private void OnInterstitialFailedEvent(string adUnitId, int errorCode)
         interstitialStatusText.text = "Failed load: " + errorCode + "\nRetrying in 3s...";
         Debug.Log("Interstitial failed to load with error code: " + errorCode);
         
-        // Interstitial ad failed to load. We recommend retrying with exponentially higher delays.
+        // Interstitial ad failed to load. We recommend retrying with exponentially higher delays up to a maximum delay.

```

Comment:
Might be worth specifying the actual delay we currently have which is 64 seconds since `double retryDelay = Math.Pow(2, Math.Min(6, interstitialRetryAttempt));` may not be blatantly obvious to a lot of publishers. Applies to the other PRs as well.

Reply:
Do you mean something like this?
`// Interstitial ad failed to load. We recommend retrying with exponentially higher delays up to a maximum delay (in this case 64 seconds).`

Reply:
That works

Reply:
changed in all PRs